packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html

```diff
@@ -253,43 +253,139 @@
                     data-term="{{:: 'pci_notebooks_general_information_resources_quantity' | translate }}"
                 >
                     <oui-tile-description>
-                        <span
-                            data-ng-if="$ctrl.flavor.gpuInformation"
-                            data-ng-bind=":: $ctrl.notebook.gpu + $ctrl.NOTEBOOK_MULTIPLY_SIGN +  ($ctrl.notebook.flavor | uppercase)"
-                        ></span>
-                        <span
-                            data-ng-if="!$ctrl.flavor.gpuInformation"
-                            data-ng-bind=":: $ctrl.notebook.cpu + $ctrl.NOTEBOOK_MULTIPLY_SIGN +  ($ctrl.notebook.flavor | uppercase)"
-                        ></span>
+                        <!-- GPU-->
+                        <span data-ng-if="$ctrl.flavor.gpuInformation">
+                            <p
```

Comment:
**issue(blocking):** Element `p` not allowed as child of element `span`

Reply:
Changed for `div`: https://github.com/ovh/manager/pull/6676/commits/2db9db3ca6d5953672b68856aa671652603dc66e